test/blocks/ost/ost.test.html.js

```diff
@@ -306,3 +306,89 @@ describe('OST: merch link creation', () => {
     });
   });
 });
+describe('OST: toggle switches', () => {
+  it('change toggle switch landscape from PUBLISHED to DRAFT', async () => {
+    const el = document.createElement('div');
+    const ostEnv = { landscape: 'PUBLISHED' };
+    const windowObj = { location: {} };
+    const cbs = addToggleSwitches(el, ostEnv, true, windowObj);
```

Comment:
Can we avoid passing the window object as a function parameter?
I think it should be possible to test without passing the `windowObj`, any reason why `expect(window.location.href).to.equal(expectedUrl);` would not work?

Nit: please separate every `it(...)` with a new line, to have it consistent with the rest of the files